libs/designsystem/src/lib/components/charts/shared/chart-js-service/test-utils.ts

```diff
@@ -1,6 +1,6 @@
 import { AnnotationTypeRegistry } from 'chartjs-plugin-annotation';
 
-import { ChartTypesConfig } from '../chart.types';
+import { ChartTypesConfig } from '../../';
```

Comment:
Same comment as above. Still not a component though.

Reply:
Same answer as above 😎